packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx

```diff
@@ -99,7 +99,7 @@ export interface TransactionReviewOutputListProps {
     actionText: TranslationKey;
     isSending?: boolean;
     setIsSending?: () => void;
-    ethereumStakeType?: StakeType;
+    stakeType?: StakeType;
```

Comment:
I can see bunch of refactoring looking changes across  this PR. don't you guys want to review them separately?